provisioning/lib/helpers

```diff
@@ -0,0 +1,26 @@
+# Usage: error_out <message> <error_code>
+error_out() {
+  echo "ERROR: ${1}"
+  exit $2
+}
+
+# Usage: get_hostname <ip address>
+get_hostnames() {
```

Comment:
@etsauer i.e.: make the domain $2, so the function can be called with get_hostnames \<ip\> \<base_domain\>. 

To that point, I'm also wondering if the hardcoded names of "master" and "node[0-9]*" should also be made a parameter. Just trying to make the function a bit more re-usable.